resources/schemas/dbscripts/postgresql/targetedms-25.001-25.002.sql

```diff
@@ -0,0 +1,69 @@
+CREATE TABLE targetedms.rateType
+(
+    Id              SERIAL NOT NULL ,
+    name            varchar(100) NOT NULL,
+    description     text DEFAULT NULL,
+    setupFee        decimal(5,2) DEFAULT 0.00,
+
+    Container       entityid NOT NULL,
+    Created         TIMESTAMP,
+    CreatedBy       USERID,
+    Modified        TIMESTAMP,
+    ModifiedBy      USERID,
+
+    CONSTRAINT PK_rateType PRIMARY KEY (Id)
+);
+CREATE INDEX IDX_rateType_Container ON targetedms.rateType(container);
+
+
+CREATE TABLE targetedms.instrumentRate
+(
+    Id              SERIAL NOT NULL ,
+    instrument      integer NOT NULL,
+    rateType        integer NOT NULL,
+    fee             decimal(7,2) NOT NULL,
+
+    Container       entityid NOT NULL,
+    Created         TIMESTAMP,
+    CreatedBy       USERID,
+    Modified        TIMESTAMP,
+    ModifiedBy      USERID,
+
+    CONSTRAINT PK_instrumentRate PRIMARY KEY (Id),
+    CONSTRAINT FK_instrumentRate_instrument FOREIGN KEY (instrument) REFERENCES targetedms.msInstrument(id),
+    CONSTRAINT FK_instrumentRate_rateType FOREIGN KEY (rateType) REFERENCES targetedms.rateType(Id)
+);
+CREATE INDEX IDX_instrumentRate_Instrument ON targetedms.instrumentRate(instrument);
+CREATE INDEX IDX_instrumentRate_RateType ON targetedms.instrumentRate(rateType);
+CREATE INDEX IDX_instrumentRate_Container ON targetedms.instrumentRate(container);
+
+CREATE TABLE targetedms.instrumentUsagePayment
+(
+    Id                          SERIAL NOT NULL ,
+    instrumentScheduleId        integer NOT NULL,
+    paymentMethod               integer NOT NULL,
+    percentPayment              decimal(5,2) NOT NULL,
+
+    Container                   entityid NOT NULL,
```

Comment:
This can be a followup commit, but I generally prefer to avoid putting `Container` on these child tables. Instead, the child tables can filter based on a join to the parent table. This is how the other tables in the targetedms schema handle containers.